lib/auth/grpcserver_test.go

```diff
@@ -371,55 +387,46 @@ func TestMFADeviceManagement(t *testing.T) {
 			desc: "delete TOTP device by name",
 			opts: mfaDeleteTestOpts{
 				deviceName:  devs.TOTPName,
-				authHandler: devs.totpAuthHandler,
+				authHandler: lastDeviceWebAuthnHandler,
 				checkErr:    require.NoError,
 			},
 		},
 		{
-			desc: "delete pwdless device by name",
+			desc: "delete webauthn device by name",
 			opts: mfaDeleteTestOpts{
-				deviceName:  pwdlessDevName,
-				authHandler: devs.webAuthHandler,
+				deviceName:  devs.WebName,
+				authHandler: lastDeviceWebAuthnHandler,
 				checkErr:    require.NoError,
 			},
 		},
 		{
-			desc: "delete last passwordless device fails",
```

Comment:
Removed because this is covered by TestDeletingLastPasswordlessDevice, correct?

Reply:
Yes